modules/nf-core/propr/logratio/meta.yml

```diff
@@ -0,0 +1,70 @@
+name: "propr_logratio"
+description: ALR/CLR transform the data matrix
+keywords:
+  - alr
+  - clr
+  - logratio
+  - boxcox
+  - transformation
+  - propr
+
+tools:
+  - "propr":
+      description: "Logratio methods for omics data"
+      homepage: "https://github.com/tpq/propr"
+      documentation: "https://rdrr.io/cran/propr/man/propr.html"
+      tool_dev_url: "https://github.com/tpq/propr"
+      doi: "10.1038/s41598-017-16520-0"
+      licence: ["GPL-2"]
+
+input:
+  - meta:
+      type: map
+      description: |
+        Groovy Map containing sample information.
+        This can be used at the workflow level to pass optional parameters to the module.
+        [id: 'test', ...]
+  - count:
+      type: file
+      description: |
+        Count matrix, where rows = variables or genes, columns = samples or cells.
+        This matrix should not contain zeros. Otherwise, they will be first replaced by the minimun value.
+        You may want to handle the zeros with a different method beforehand.
+      pattern: "*.{csv,tsv}"
+  - transformation:
```

Comment:
You should remove anything from this file that's not in the inputs.

Reply:
even though if the arguments like transformation, alpha, etc would be useful for the user to declare ext.args?
Do you suggest another way or place to put this information for the users?

Reply:
Yep, current policy is that we only document things in channels, with those being the minimal things required to operate a module. So if a value is mandatory it goes in via channels (like the Salmon example I indicated earlier) and gets documented in the meta.yml. Everything else is considered optional, goes in via ext.args, and we defer to the tool's native documentation (to avoid maintaining duplicate info here).